File: bench/cases/split-only-100k/main.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	var total int64 = 0
	var n int64 = 100_000
	for i := int64(0); i < n; i++ {
		parts := strings.Split("3 4 + 2 * 5 +", " ")
		total += int64(len(parts))
	}
	fmt.Println(total)
}

File: bench/cases/split-only-100k/main.rs
fn main() {
    let mut total: i64 = 0;
    let n: i64 = 100_000;
    for _ in 0..n {
        let parts: Vec<&str> = "3 4 + 2 * 5 +".split(' ').collect();
        total += parts.len() as i64;
    }
    println!("{}", total);
}
